velox/benchmarks/FilterBenchmark.cpp

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Copying an existing benchmark for now, just to proof the concept.

Reply:
Where do we log the benchmark results - i assume thats something we still have to work on ?

Reply:
Folly/benchmarks handles it. It can output it to a json file.